packages/experimental/Checkbox/src/Checkbox.styling.ts

```diff
@@ -19,54 +18,52 @@ export const stylingSettings: UseStylingOptions<CheckboxProps, CheckboxSlotProps
   states: checkboxStates,
   slotProps: {
     root: buildProps(
-      (tokens: CheckboxTokens) => ({
```

Comment:
I don't think these changes are from running 'yarn prettier'? Is this supposed to be here? If they are, maybe add a quick comment to the description of the PR outlining the change just to avoid confusion :)

Reply:
🤦‍♀️must've checked the branch out from the wrong place. Sorry. I'll probably redo this PR.